ValveResourceFormat/CompiledShader/FILETYPE.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ValveResourceFormat.ShaderParser
+{
+    public enum FILETYPE {
+        unknown, any, features_file, vs_file, ps_file, gs_file, psrs_file
```

Comment:
this would be nicer to have proper names, like `PixelShader` etc (and one per line pls)

Reply:
Noted your remarks, thanks (I made the changes). I'll update with the lzma decompression (solved) and hlsl parsing (might take a little longer)